app/hedge.py

```diff
@@ -1,58 +1,159 @@
-import output
-import post
-import power
-import subprocess
-import thermo
+#  =======================================
+# Import Statements
+#  =======================================
+
 import weight
-from time import sleep
-from datetime import datetime
+import pir
+import time
+import rfid
+import thermo
+from multiprocessing import Process
+import client
+import output
 import glob
 import os
-import led
-import rfid
-
+import pysftp
```

Comment:
Remove pysftp and let pizero post to pi/Videos

Reply:
We've been having a look at this, almost got it working